README.md

```diff
@@ -1,56 +1,121 @@
-![](https://github.com/OpenLiberty/open-liberty/blob/master/logos/logo_horizontal_light_navy.png)
+<!-- PROJECT LOGO -->
+
+<p align="center">
+  <a href="https://openliberty.io/">
+    <img src="https://openliberty.io/img/spaceship.svg" alt="Logo">
+  </a>
+</p>
+<p align="center">
+  <a href="https://openliberty.io/">
+    <img src="https://github.com/OpenLiberty/open-liberty/blob/master/logos/logo_horizontal_light_navy.png" alt="title" width="400">
+  </a>
+</p>
+<br />
 
-# OpenLiberty
 
-[![Twitter](https://img.shields.io/twitter/follow/openlibertyio.svg?style=social&label=Follow)](https://twitter.com/OpenLibertyIO)
-[![License](https://img.shields.io/badge/License-EPL%201.0-green.svg)](https://opensource.org/licenses/EPL-1.0)
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](CONTRIBUTING.md)
 [![Maven Central](https://img.shields.io/maven-central/v/io.openliberty/openliberty-runtime.svg?label=Maven%20Central)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22io.openliberty%22%20a%3A%22openliberty-runtime%22)
+[![Docker Pulls](https://img.shields.io/docker/pulls/_/open-liberty.svg?color=yellow)](https://hub.docker.com/_/open-liberty)
+[![Website](https://img.shields.io/badge/website-live-purple.svg)](https://openliberty.io/)
+[![Stack Overflow](https://img.shields.io/badge/find-answers-blue.svg)](https://stackoverflow.com/questions/tagged/open-liberty)
+[![Groups.io](https://img.shields.io/badge/ask-groups.io-orange.svg)](https://groups.io/g/openliberty)
+[![License](https://img.shields.io/badge/License-EPL%201.0-green.svg)](https://opensource.org/licenses/EPL-1.0)
+[![Twitter](https://img.shields.io/twitter/follow/openlibertyio.svg?style=social&label=Follow)](https://twitter.com/OpenLibertyIO)
 
 # Summary
-Open Liberty is a highly composable, fast to start, dynamic application server runtime environment.
-
-# Table of Contents
-* [Getting Started](https://github.com/OpenLiberty/open-liberty#getting-started)
-* [Contribute to Open Liberty](https://github.com/OpenLiberty/open-liberty#contribute-to-open-liberty)
-* [Community](https://github.com/OpenLiberty/open-liberty#community)
+A lightweight open framework for building fast and efficient cloud-native Java microservices:
```

Comment:
is it correct to say that OpenLiberty is a "framework"? I've not seen this phrasing used anywhere else

Reply:
yeah this is straight from the Homepage redesign - keeping for now - want to stay in sync.